bril-txt/briltxt.py

```diff
@@ -210,6 +222,12 @@ def float(self, items):
     def nullptr(self, items):
         return 0
 
+    def char(self, items):
+        value = str(items[0])[1:-1] # Strip `'`.
```

Comment:
```suggestion
        value = str(items[0])[1:-1]  # Strip `'`.
```

Just a tiny PEP8 style thing.